playbooks/roles/airship-deploy-ucp/defaults/main.yml

```diff
@@ -2,3 +2,5 @@
 scale_profile: "{{ lookup('env','scale_profile') | default('minimal', true) }}"
 airship_pegleg_image: "{{ suse_airship_registry_location }}/airshipit/pegleg:{{ suse_airship_components_image_tag }}"
 suse_osh_image_version: "pike"
+run_tests: true
+test_timeout: 1800
```

Comment:
can we please increase the default timeout to 2700? For the Nova chart, we changed the upstream default to 2700. Since we use a global value for all charts, we should pick the largest value of all chart.